### ISSUE TITLE
Polyfills are deprecated

### ISSUE THREAD

**Issue by M4SX5** [sprain/php-swiss-qr-bill#141]
I successfully set up the qr bill for my application. Nevertheless, I always get the following error message:

`Deprecated: Since symfony/intl 5.3: Polyfills are deprecated, try running "composer require symfony/polyfill-intl-icu ^1.21" instead.`

 Is there a possibility to upgrade the dependent libraries?

**Comment by M4SX5** [comment]
Hi Sprain
Thank you very much for your answer. I'm working with different versions of php 8 (currently 8.0.10). The deprecated warnign occurs with all versions. There are several other dependencies in my project. Below you find a list of the dependencies: 

"payrexx/payrexx": "dev-master",
"phpmailer/phpmailer": "^6.1",
"components/jquery": "3.5.*",
"nikic/fast-route": "^1.3",
"components/jqueryui": "^1.12",
"jumbojett/openid-connect-php": "^0.9.1",
"tecnickcom/tcpdf": "^6.3",
"spatie/schema-org": "^3.3",
"guzzlehttp/guzzle": "^7.3",
"microsoft/azure-storage-blob": "^1.5",
"sendinblue/api-v3-sdk": "7.x.x",
"sprain/swiss-qr-bill": "^3.1"

**Comment by kohlerdominik** [comment]
Hi @M4SX5 

Thanks for this report. From your stack trace it looks like that this deprecation warning is triggered indirectly in `symfony/validator`. But `symfony/validator` does require neither `symfony/intl` nor `symfony/polyfill-intl-icu`, looks like this is up to the library using validator-methods which depend on this libraries.

So it might be true that we should add `symfony/polyfill-intl-icu` as dependeny, as we rely on it by using the `CountryValidator` which depends on `Symfony\Component\Intl\Countries::exists()` which then utilizes its `Symfony\Component\Intl\Locale`. This extends `\Locale`, which if triggers it's own polyfill in case the [php-intl](https://www.php.net/manual/en/book.intl.php) extension is not installed.

TLDR:
If you just `composer require symfony/polyfill-intl-icu ^1.21` in your project, the deprecation warning should disapear. Can you confirm this?

**Comment by M4SX5** [comment]
Hi @kohlerdominik 

Thank you for your answer. I just tried your suggestion and can confirm, that when I add polyfill-intl-icu as a dependency in my project (`composer require symfony/polyfill-intl-icu ^1.21`) the deprecation warning disapears.
